assets/prefabs/recipe/arms/Copper/RecipeCopperaxe.prefab

```diff
@@ -0,0 +1,18 @@
+{
+    "ProcessDefinition": {
+        "processType": "EquipmentSmithing:ForgingTier3Process"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "EquipmentSmithing:CopperAxe",
+
+        "recipeComponents": ["3*Smithing:CopperBar"],
+        "recipeTools": ["1*forgeHammer"],
+        "requiredTemperature": 400,
```

Comment:
Are those temperatures taken into account, and should they be somewhat realistic? In that case, according to [[1]] the temperature for copper should be around 900 °C.

[1]: https://en.wikipedia.org/wiki/Forging_temperature